docs/source/en/using-diffusers/tips.md

```diff
@@ -0,0 +1,167 @@
+
+```py
+from diffusers import StableDiffusionPipeline, DPMSolverMultistepScheduler
+import torch
+
+pipeline = StableDiffusionPipeline.from_single_file(
+    "https://huggingface.co/gsdf/Counterfeit-V3.0/blob/main/Counterfeit-V3.0_fix_fp16.safetensors",
+    torch_dtype=torch.float16,
+)
+pipeline.load_textual_inversion(
+    "embed/EasyNegative",
+    weight_name="EasyNegative.safetensors",
+    token="EasyNegative"
+)
+```
+
+2. This is optional, but if there is a specific style (typically a LoRA adapter) you want to apply to the images, download the weights and use the [`load_lora_weights`] method to add it to the pipeline. This example uses the [Dungeon Meshi Marcille Character Lora](https://civitai.com/models/106199/dungeon-meshi-marcille-character-lora).
+
+```py
+!wget https://civitai.com/api/download/models/114049 -O marcille.safetensors
+pipeline.load_lora_weights('.', weight_name="marcille.safetensors")
+```
+
+3. Load a scheduler and set `use_karras_sigmas=True` to use the DPM++ 2M Karras scheduler (take a look at this [scheduler table](../api/schedulers/overview.) to find the A1111 equivalent scheduler in Diffusers).
+
+```py
+pipeline.scheduler = DPMSolverMultistepScheduler.from_config(pipeline.scheduler.config)
+pipeline.scheduler.config.use_karras_sigmas=True
+pipeline.to('cuda')
+```
+
+4. Create your prompt and negative prompts, and remember to use the trigger words for this specific LoRA adapter (`dmarci`) and embeddings (`EasyNegative`). It is also important to set the:
+
+    - `lora_scale` parameter to control how to scale the output with the LoRA weights.
+    - `clip_skip` parameter to specify the layers of the CLIP model to use. This parameter is especially important for anime checkpoints because it controls how closely aligned the text prompt and image are. A higher `clip_skip` value produces more abstract images.
+
+```py
+generator = torch.Generator("cpu").manual_seed(0)
+
+prompt = "dmarci, masterpiece, best quality, 1girl, solo, marcillessa, red choker, detailed and beautiful eyes, (cowboy shot:1.2), HAPPY, walking, jumping,(Turtleneck_sweater:1.4), (Leather_skirt:1.3)"
+negative_prompt = "EasyNegative, (worst quality, low quality, bad quality, normal quality:2), logo, text, blurry, low quality, bad anatomy, lowres, normal quality, monochrome, grayscale, worstquality, signature, watermark, cropped, bad proportions, out of focus, username, bad body, long body, (fat:1.2), long neck, deformed, mutated, mutation, ugly, disfigured, poorly drawn face, skin blemishes, skin spots, acnes, missing limb, malformed limbs, floating limbs, disconnected limbs, extra limb, extra arms, mutated hands, poorly drawn hands, malformed hands, mutated hands and fingers, bad hands, missing fingers, fused fingers, too many fingers, extra legs, bad feet, backlighting"
```

Comment:
just a heads up, this style of prompting doesn't work with vanilla diffusers, this needs the `lpw` community pipelines , e.g.: [examples/community/lpw_stable_diffusion.py](https://github.com/huggingface/diffusers/blob/8f2d13c684e282b258e5fbc73744eda4dc5f3cb6/examples/community/lpw_stable_diffusion.py)  but right now you can't use them with `from_single_file` and they don't have the `clip_skip` argument, there are PRs for both of this issues though.

So before that it would be more accurate to remove the weighted prompting, like `(cowboy shot:1.2) -> cowboy shot` and limit the tokens to 75.